client/release.go

```diff
@@ -155,26 +155,18 @@ func (c *Client) PromoteRelease(appID string, appType string, sequence int64, la
 	return errors.New("unknown app type")
 }
 
-// yamlOrJSON is either a single ship yaml file, or a serialized JSON object describing a yaml-dir, created by readYAMLDir()
+// data is either a single ship yaml file, or a serialized JSON object describing a yaml-dir, created by readYAMLDir()
```

Comment:
what was the reasoning behind this naming change? I had switched it from `data` because `data` was kind of unclear.

Reply:
for kots, it is no longer a yaml or a json string, it is the tarred yaml dir containing all of the manifests as a []byte.

Reply:
okay, wanna update the comment? I'm still trying to understand how this is gonna work for ship

Reply:
@salahalsaleh it appears I removed all support for ship linting in this PR 🤦  https://github.com/replicatedhq/replicated/commit/bb7463d1145be607b4eec4f8f8d4d377398e6999

Reply:
Aha, so we do not support ship linting anymore? I can clean all of this up????? :)

Reply:
yeah lets